Burn Bryte by Roll20/BurnBryte.html

```diff
@@ -1149,295 +1129,231 @@ <h3 data-i18n="special-a">Special Abilities</h3>
 //        setAttrs({social: newSocial});
 //    });
 //});
-
 //Initiative Toggle
 on("change:initiative_toggle", function() {
```

Comment:
On this line in the function you get user eventinfo to remove the extra step of doing getAttrs.

For example if you do `function(eventinfo)` on this line then you can remove the getAttrs and just do `const toggle = (eventinfo.newValue);`